docs/DeveloperGuide.md

```diff
@@ -574,91 +574,19 @@ The sequence diagram is as follows:
 
 The multiple commands executed will return a `CommandResult` which contains a list of feedback messages of the results executed combined all together and returned as `feedbackToUser`
 
+### \[Proposed\] Tag Finding Feature
 
-### \[Proposed\] Undo/redo feature
+The proposed find tag feature is similar to `findperson` where the user can search for tags from the specified keywords. This feature was not implemented as our target user need not maintain a large number of tabs (compared to persons and appointments). However, if this feature is required in the future, the developer simply needs to implement the `Logic` components as the required components in `Model` already exists.
```

Comment:
Spelling error here.
```suggestion
The proposed find tag feature is similar to `findperson` where the user can search for tags from the specified keywords. This feature was not implemented as our target user need not maintain a large number of tags (compared to persons and appointments). However, if this feature is required in the future, the developer simply needs to implement the `Logic` components as the required components in `Model` already exists.
```

Reply:
Fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/337/commits/fc832993c54ab353a250ff5bc72c4396a4bfa173